README.md

```diff
@@ -44,15 +50,15 @@ curl -w"\n" -X POST "localhost:11015/v3/namespaces/default/artifacts/word-count-
    --data-binary @/PATH/TO/wordcount-1.0.0.jar
 ```
 
-> Note that the Spark application is now turned into a CDAP plugin without any modification to your code. Your Spark application will be running within a wrapped CDAP application as a Plugin. ```Artifact-Extends``` header specifies the CDAP application this Spark application will be a plugin for. Also, note that in the configuration above you specify the ```className``` name that CDAP should use to invoke your Spark application. 
+> Note that the Spark application is now turned into a CDAP plugin without any modification to the Spark code. ```Artifact-Extends``` header specifies the CDAP application that will run the Spark code. The driver class name for the Spark application is specified using the ```className``` parameter. 
```

Comment:
I think we can say `Main class name for the Spark application is specified using the` instead of driver class name. Same terminology is used here - http://spark.apache.org/docs/latest/submitting-applications.html